HISTORY.rst

```diff
@@ -1,8 +1,14 @@
 *Bug tracker at https://github.com/giampaolo/psutil/issues*
 
+5.9.3 (IN DEVELOPMENT)
+======================
```

Comment:
@giampaolo I saw you just released `5.9.2`.  Is that entry okay like this?

Reply:
yup, that's fine